Explorer/Assets/DCL/CharacterTriggerArea/CharacterTriggerArea.cs

```diff
@@ -7,49 +7,35 @@ namespace DCL.CharacterTriggerArea
     public class CharacterTriggerArea : MonoBehaviour, IDisposable
     {
         [field: SerializeField] public BoxCollider BoxCollider { get; private set; }
+        public HashSet<Transform> EnteredThisFrame;
+        public HashSet<Transform> ExitedThisFrame;
```

Comment:
Encapsulation is broken, anyone could assign these HashSet from anywhere. Take a look on the proposal on this branch `feat/sdk-camera-mode-area_refactor-proposal-1`